lib/src/dto/contributor.payload.ts

```diff
@@ -7,5 +6,5 @@ export type ContributorPayload = {
     displayName: string;
     url: string;
   }
-  type: RoleSetContributorType
+  type: string
```

Comment:
Are you doing this because we ultimately want to get rid of the lib or other reasons? 
In my opinion we should swap it with another narrow type then the generic `string` type. There are still type assertions all around the code using the lib, so I don't believe that helps more than it hurts.

Reply:
Yes to remove dependencies on enums / types from lib.